service/devdoc/iothub_client_requirements.md

```diff
@@ -109,32 +109,32 @@ The `send` method sends a cloud-to-device message to the service, intended for d
 
 **SRS_NODE_IOTHUB_CLIENT_16_030: [** The `send` method shall not throw if the `done` callback is falsy. **]**
 
-sendToModuleInput(deviceId: string, moduleId: string, inputName: string, message: Message | Message.BufferConvertible, done?: Callback<results.MessageEnqueued>): void;
-The `sendToModuleInput` method sends a message to the service, intended for delivery to the given input on the given module on the given device
+sendToModule(deviceId: string, moduleId: string, message: Message | Message.BufferConvertible, done?: Callback<results.MessageEnqueued>): void;
```

Comment:
>sendToModule [](start = 0, length = 12)

thinking out loud (sortof) ; if we're naming this "sendToModule" should we deprecate "send" and rename it "sendToDevice" ?
What are the other SDKs calling these APIs?